src/plugins/intel_cpu/src/emitters/snippets/cpu_runtime_configurator.cpp

```diff
@@ -41,25 +43,22 @@ CPURuntimeConfigurator::CPURuntimeConfigurator() : ov::snippets::RuntimeConfigur
 
 void CPURuntimeConfigurator::initialization(const ov::snippets::lowered::LinearIRCPtr& linear_ir) {
     RuntimeConfigurator::initialization(linear_ir);
-    if (linear_ir->is_dynamic()) {
-        loopPortsAdjuster = BrgemmCopyBLoopPortsAdjuster(linear_ir);
-    }
+#ifndef OPENVINO_ARCH_ARM64
+    if (linear_ir->is_dynamic())
```

Comment:
Should check `linear_ir->is_dynamic()` be in `BrgemmCopyBLoopPortsAdjuster::applicable()`?

Reply:
Why not, I moved this check there and cleaned up configurator's code. The same was done for MHA parallel work amount optimizer

Reply:
Now this's better, thanks!